sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/compatibility/AzureSpringBootVersionVerifier.java

```diff
@@ -113,9 +119,10 @@ private boolean springBootVersionMatches() {
 
     private boolean matchSpringBootVersionFromManifest(String acceptedVersion) throws FileNotFoundException {
         String version = this.getVersionFromManifest();
-        LOGGER.debug("Version found in Boot manifest [{}]", version);
+        LOGGER.debug("Currently running on Spring Boot version [{}], trying to match it in Spring Cloud Azure "
+            + "Supported Versions [{}].", version, acceptedVersion);
         if (!StringUtils.hasText(version)) {
-            LOGGER.info("Cannot check Boot version from manifest");
+            LOGGER.info("Cannot check current Spring Boot version from Spring Cloud Azure supported versions.");
```

Comment:
This log doesn't make sense. The `version` returns from the `getVersionFromManifest()` method, but the log is `from Spring Cloud Azure supported versions.` What does this mean?